src/backend_model.h

```diff
@@ -99,10 +124,21 @@ class TritonModel : public Model {
       const std::shared_ptr<LocalizedPath>& localized_model_dir,
       const std::shared_ptr<TritonBackend>& backend,
       const double min_compute_capability, const int64_t version,
-      const inference::ModelConfig& config, const bool auto_complete_config);
+      const inference::ModelConfig& config, const bool auto_complete_config,
+      const triton::common::BackendCmdlineConfigMap& backend_cmdline_config_map,
+      const triton::common::HostPolicyCmdlineConfigMap& host_policy_map);
 
-  // Set the scheduler based on the model configuration. The scheduler
-  // can only be set once for a backend.
+  // Replace the foreground instances with background instances.
+  Status CommitInstances();
+
+  // Set the 'device_blocking_' flag.
+  Status SetExecutionPolicy(const inference::ModelConfig& model_config);
```

Comment:
GetExecutionPolicy()

Reply:
Updated: [Rename SetExecutionPolicy to GetExecutionPolicy](https://github.com/triton-inference-server/core/pull/189/commits/fe6eab259c11d2f078b716be515ff2e32d0e322c)